docs/administering/sniproxy.md

```diff
@@ -0,0 +1,146 @@
+# Sharing the HTTPS Port with sniproxy
+
+The purpose of this tutorial is to set up sniproxy so it’s possible to use regular https:// URLs for your main web server, like nginx or Apache, as well as Sandstorm, which has its own HTTPS handling. To make that work, they will need to share port 443.
+
+## Introduction
+
+The purpose of this tutorial is to set up sniproxy so it's possible to use Sandstorm-verified TLS encryption while coexisting with another web server that also uses TLS and share port 443 with it.
+
+The main reason is to allow users to connect with your Sandstorm instance in the standard HTTPS port (443) and keep using that port also for any other web apps.
+
+This action is going to change the BASE_URL of Sandstorm, presumably, so you'll end up erasing the OAuth configuration. To fix that, you will need to follow the "How do I log in, if there's a problem with logging in via the web?" section of [the FAQ](faq.md).
+
+This document assumes the server is running Debian or one of its derivatives (e.g. Ubuntu). If your server is not, you might need to adjust some steps. Note that there will be some downtime in this process so you might want to do it when your server is not very busy.
+
+## Install sniproxy
+
+If you're lucky, the package `sniproxy` might be present on your GNU/Linux distro; otherwise, you'll have to install it yourself. Follow the instructions at [the sniproxy homepage](https://github.com/dlundquist/sniproxy) for your operating system and install it.
+
+In this example, on Ubuntu 10.04, the instructions were:
```

Comment:
Should we maybe update this for a new version? This is literally a decade-old distro. It looks like Debian and Ubuntu both ship packages for sniproxy now, so there's no real reason to build from source, and with the source method the user won't get automatic updates with the rest of their distro.

I imagine these steps can all be replaced by `sudo apt-get install sniproxy` or something.

Reply:
The paragraph above does specify it might be in your distro. I guess we should make that sound less "hopeful" and more likely. And then just remove this, as the sniproxy homepage linked above is stated to have the correct instructions if a manual install is needed. We shouldn't be bothering to keep updated instructions. (I noticed how old this distro was, but I didn't want to modify it from "the last thing someone knows worked". Getting rid of it simplifies this problem.)

Reply:
Ok, yeah, let's just get rid of it.